activities/finance-management/authorization.md

```diff
@@ -0,0 +1,12 @@
+---
+type: Guide
+explains: how authorization is managed at the Foundation
+---
+
+# Authorization
+
+Board members – two board members can decide on payments above € 50.000. [See bylaws.](../../organization/articles-of-association.md)
```

Comment:
In American/British English, you don't usually put a space between a currency symbol and currency amount.